schema/iTime.sql

```diff
@@ -0,0 +1,98 @@
+--
```

Comment:
I will have to look into this, we could store the schema somewhere. 
Not sure how yet.